### PR TITLE
Hacken Audit Fixes

### PR DESCRIPTION
**Pull request by narayanprusty**
## Description

Fixes issues reported in hacken audit

## Checklist

- [ ] I have updated the documentation to account for the changes in the code.
- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [ ] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

**Review by UrAvgDeveloper — APPROVED** [review]
LGTM. No major changes.

File: contracts/PriceOracle.sol [hunk 1]
```diff
@@ -1,9 +1,7 @@
-pragma solidity >=0.5.16;
+// SPDX-License-Identifier: UNLICENSED
```
Comment (kkirka) [inline contracts/PriceOracle.sol:1]:
```suggestion
// SPDX-License-Identifier: BSD-3-Clause
// SPDX-FileCopyrightText: 2020 Compound Labs, Inc.
// SPDX-FileCopyrightText: 2022 Venus
```

File: contracts/ResilientOracle.sol [hunk 2]
```diff
@@ -1,6 +1,5 @@
 // SPDX-License-Identifier: UNLICENSED
```
Comment (kkirka) [inline contracts/ResilientOracle.sol:1]:
```suggestion
// SPDX-License-Identifier: BSD-3-Clause
// SPDX-FileCopyrightText: 2022 Venus
```

File: contracts/interfaces/PythInterface.sol [hunk 3]
```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: Apache-2.0
```
Comment (kkirka) [inline contracts/interfaces/PythInterface.sol:1]:
```suggestion
// SPDX-License-Identifier: Apache-2.0
// SPDX-FileCopyrightText: 2021 Pyth Data Foundation
```

File: contracts/oracles/ChainlinkOracle.sol [hunk 4]
```diff
@@ -180,7 +177,7 @@ contract ChainlinkOracle is OwnableUpgradeable, OracleInterface {
      */
     function setTokenConfigs(TokenConfig[] memory tokenConfigs_) external onlyOwner {
         require(tokenConfigs_.length > 0, "length can't be 0");
-        for (uint256 i = 0; i < tokenConfigs_.length; i++) {
+        for (uint256 i; i < tokenConfigs_.length; ++i) {
```
Comment (kkirka) [inline contracts/oracles/ChainlinkOracle.sol:180]: Maybe cache the length as well?
(Here and in other contracts)

```suggestion
        uint256 numTokenConfigs = tokenConfigs_.length;
        for (uint256 i; i < numTokenConfigs; ++i) {
```

**Review by kkirka — APPROVED** [review]
Left some examples of the correct SPDX headers and a comment about a possible gas optimization, but it's minor. Overall LGTM

**Review by defcon022 — APPROVED** [review]

**Pull request merged**